rust/kcl-lib/src/execution/geometry.rs

```diff
@@ -881,6 +881,19 @@ pub enum Path {
         #[ts(type = "[number, number]")]
         p3: [f64; 2],
     },
+    ArcThreePoint {
+        #[serde(flatten)]
+        base: BasePath,
+        /// Point 1 of the circle
+        #[ts(type = "[number, number]")]
+        p1: [f64; 2],
+        /// Point 2 of the circle
+        #[ts(type = "[number, number]")]
+        p2: [f64; 2],
+        /// Point 3 of the circle
+        #[ts(type = "[number, number]")]
```

Comment:
```suggestion
        /// Point 1 of the arc
        #[ts(type = "[number, number]")]
        p1: [f64; 2],
        /// Point 2 of the arc
        #[ts(type = "[number, number]")]
        p2: [f64; 2],
        /// Point 3 of the arc
        #[ts(type = "[number, number]")]
```

I'm a little confused why Rust changes are needed, I thought they were covered by #4485?

Reply:
stdlib added to rust almost never is 100% right ready for point and click (for segments that is at least.)

Reply:
61caf59d09007a10816657b6695125f221f6b8a1